server/services/damsService/utils.js

```diff
@@ -0,0 +1,129 @@
+const { ui } = require("../../../src/shared/config");
```

Comment:
This file is not actually any real changes - I just moved the functions out from [here](https://github.com/BarnesFoundation/Barnes-Online-Collection/pull/291/files#diff-d239a4782487090e7fcd74b1459b34467adc0e6349638f39672eab8332bbf102L22-L206) into this file for better organization